server/src/parser/ast.ts

```diff
@@ -42,10 +42,11 @@ export default class AST {
     };
   }
 
-  static ifStatement(clauses: GeneralIfClause[]): IfStatement {
+  static ifStatement(clauses: GeneralIfClause[], oneLine: boolean): IfStatement {
```

Comment:
@japhib Is this `oneLine` extra param something that should be no longer added in the most recent version of code from `master`? It was added in a 6195b55f133759ed13212047ff478f2bd3b2a806 commit from `japhib-formatter` which based on `0.3.1` version of `pico8-ls`, and I don't know if it is needed for formatter only or should be added to `0.4.8` even without formatter 🤔

Reply:
I think we will need it. Right now in `Parser.parseIfStatement()` it knows how to parse a one-line if statement, but internally in the AST it represents it the exact same as a regular full `if then ...` statement. So we'll need to add this `oneLine` flag if we want the formatter to know it should emit a one-line if statement rather than `if then ... end`.